Readme.md

```diff
@@ -24,13 +24,13 @@ You find working examples for different use-cases under the [examples](./example
 
 ## Support Matrix
 
-| Component | Consume | Publish | CloudEvents | Metrics | Custom |
-| --- | --- | --- | --- | --- | --- |
-| RabbitMQ | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | priority, dynamic routing |
-| Kafka | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | partitioning key, dynamic topic |
-| Http | (:heavy_check_mark:) | (:heavy_check_mark:) | :x: |:heavy_check_mark:| |
-| Timer | (:heavy_check_mark:) | - | :x: | :x:| |
-| SQS | (:heavy_check_mark:) | - | :x: | :x:| |
+| Component | Consume | Publish | CloudEvents | Metrics | Compression | Custom |
+| --- | --- | --- | --- | --- | --- | --- |
+| RabbitMQ | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | priority, dynamic routing |
+| Kafka | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | partitioning key, dynamic topic |
+| Http | (:heavy_check_mark:) | (:heavy_check_mark:) | :x: |:heavy_check_mark: | :x: | |
+| Timer | (:heavy_check_mark:) | - | :x: | :x: | :x: | |
+| SQS | (:heavy_check_mark:) | - | :x: | :x: | :heavy_check_mark: | |
```

Comment:
@FlorianHockmann SQS uses the same Consumer which supports compression. However, it seems there is currently no way to set the `compression-type` attribute, because we don't have an SQS-compatible publisher.

Reply:
Right, my take would be that SQS supports compression in general, but since there is no publisher right now, we can of course only support it in the consumer. The publisher would have to use it the way we expect it.

Reply:
The support matrix states that SQS is incompatible with `MotorCloudEvent`s. Based on a quick inspection of the code, I think it only wraps the received message in a `MotorCloudEvent`, which has no attributes. Therefore I don't think it is only the publishing mechanism that has to be implemented. There would still be some work to be done to handle message attributes correctly for SQS.

Reply:
OK, then I think that we should just remove the check mark here.